File: Notification/CustomNotif/java/app/src/main/java/com/dicoding/picodiploma/customnotif/MainActivity.java
package com.dicoding.picodiploma.customnotif;

import android.content.Intent;
import androidx.appcompat.app.AppCompatActivity;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        Button buttonNotif = findViewById(R.id.button_show_notification);
        buttonNotif.setOnClickListener(view ->
                startService(new Intent(MainActivity.this, NotificationService.class))
        );
    }
}

File: Notification/CustomNotif/kotlin/app/src/main/java/com/dicoding/picodiploma/customnotif/MainActivity.kt
package com.dicoding.picodiploma.customnotif

import android.content.Intent
import android.os.Bundle
import android.widget.Button
import androidx.appcompat.app.AppCompatActivity

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        val buttonNotif = findViewById<Button>(R.id.button_show_notification)
        buttonNotif.setOnClickListener {
            startService(Intent(this, NotificationService::class.java))
        }
    }
}

